### ISSUE TITLE
no vignettes?

### ISSUE THREAD

**Issue by greg-minshall** [nset-ornl/wbstats#20]
hi.  thanks *so* much for this!  but... :)

the `?wbstats` output includes

```
     To learn more about the wbstats package, start with the vignettes:
     ‘browseVignettes(package = "wbstats")’
```

however,

```
> browseVignettes(package = "wbstats")
No vignettes found by browseVignettes(package = "wbstats")
```

cheers!

**Comment by greg-minshall** [comment]
i should mention i'm installing from github.  i just re-did my install, and the problem persists (in *spite* of the `installing vignettes` line!).  cheers.

```
devtools::install_github("nset-ornl/wbstats", force=TRUE)
   checking for file ‘/tmp/RtmpF6SRMI/remotes4b66e7f14e14d/nset-ornl-wbstats-5501b8c/DESCRIPTIO✔  checking for file ‘/tmp/RtmpF6SRMI/remotes4b66e7f14e14d/nset-ornl-wbstats-5501b8c/DESCRIPTION’
─  preparing ‘wbstats’:
✔  checking DESCRIPTION meta-information ...
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  looking to see if a ‘data/datalist’ file should be added
─  building ‘wbstats_1.0.0.tar.gz’

Installing package into ‘/home/minshall/usr/lib/R/x86_64-pc-linux-gnu-library’
(as ‘lib’ is unspecified)
* installing *source* package ‘wbstats’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (wbstats)
```

**Comment by jpiburn** [comment]
Hi, sorry for the delay in replying... I did a lot of re-configuring when I started using `pkgdown` for the website and I must have let that slip through. Thank you for letting me know

**Comment by greg-minshall** [comment]
you're welcome.  thanks for looking at it.

('delay' -- *i* always need time to "page in" the memory cells, and that takes time, and it's nice to 'batch', i.e., to be able to amortize that cost over various work items. :)  cheers!

**Comment by jpiburn** [comment]
Hi, this issue should be resolved with the latest release, `v1.0.1` which is now available on CRAN
